protocol-definitions/custom/Custom.yaml

```diff
@@ -194,6 +194,10 @@ customTypes:
         type: BitmapIndexOptions
         nullable: true
         since: 2.0
+      - name: bTreeIndexConfig
+        type: BTreeIndexConfig
+        nullable: true
```

Comment:
should we make this non-nullable as well?

Reply:
same as BitmapIndexOptions, user may give null value